src/Components/Common/utils/Tooltip.tsx

```diff
@@ -0,0 +1,30 @@
+import clsx from "clsx";
+import { ReactNode, useState } from "react"
+
+export default function ToolTip(props : {
+    children : ReactNode
+    text : ReactNode
+    position : "TOP" | "BOTTOM" | "LEFT" | "RIGHT"
+}){
+
+    const [status, show] = useState(false);
+
+    const style = clsx({
+        "absolute bg-black/50 backdrop-blur rounded text-white transition px-2 py-1 z-50 w-[150px] text-center block" : true,
```

Comment:
You can remove `true` as it is not needed
```suggestion
        "absolute bg-black/50 backdrop-blur rounded text-white transition px-2 py-1 z-50 w-[150px] text-center block",
```

Reply:
Hey, this will be required as this is an object